### ISSUE TITLE
缺少ImageDataset_GCN_Fundus(OCTA,2PFM)类

### ISSUE THREAD

**Issue by DBook111** [Zhouyi-Zura/MaskVSC#3]
作者你好，非常感谢你们的启发性的工作。
我发现在 train_ViG.py 中缺少发三个loader类，请问这三个类和训练脚本中的三个类的区别是什么？

**Comment by STOPSUNS** [comment]
Dear Author, in your train_ViG.py code, the following section:

def train(best_seg_pretrain=None):
    print("Load Segmentation Model: %s!" % best_seg_pretrain)
    Seg_model.load_state_dict(torch.load(best_seg_pretrain))
    prev_time = time.time()
    best_dice = 0.0
    for epoch in range(opt.n_epochs):        
        # ---Dataloader---
        if opt.data_type == "Fundus":
            train_dataloader = DataLoader(
                ImageDataset_GCN_Fundus(train_path, transforms_=transforms_),
                batch_size=opt.batch_size, shuffle=True, num_workers=opt.n_cpu,)
        elif opt.data_type == "OCTA":
            train_dataloader = DataLoader(
                ImageDataset_GCN_OCTA(train_path, transforms_=transforms_),
                batch_size=opt.batch_size, shuffle=True, num_workers=opt.n_cpu,)
        elif opt.data_type == "2PFM":
            train_dataloader = DataLoader(
                ImageDataset_GCN_2PFM(train_path, transforms_=transforms_),
                batch_size=opt.batch_size, shuffle=True, num_workers=opt.n_cpu,)

refers to three GCN dataset classes (ImageDataset_GCN_Fundus, ImageDataset_GCN_OCTA, and ImageDataset_GCN_2PFM), but the corresponding code for these is not provided. Could you please provide these files or code snippets?"

**Comment by Zhouyi-Zura** [comment]
@DBook111 @STOPSUNS  Hi all, thanks for the interest on our work. I have update the 'datasets.py'.
